Documentation/build-firmware.md

```diff
@@ -127,3 +126,16 @@ dd if=firmware_fit.merged of=/dev/sdX bs=512 seek=2
 
 ### Deploying UEFI (uefi.fit) for development
 Copy `uefi.fit` over to the EFI partition on your SD card.
+
+### Updating the TAs in UEFI
+A firmware TPM TA, and UEFI authenticated variable TA, are included with EDK2. Generally, these TAs should work on any ARM32 system where OP-TEE is running, and eMMC RPMB is available.
+
+These binaries are built using OpenSSL by default but can also be built using WolfSSL (See `FTPM_FLAGS` and `AUTHVAR_FLAGS` in `common.mk`).
```

Comment:
The ARM64 binaries look to be built using WolfSSL.  Have we not ported that yet? Also the armt32 imx7 uses wolf for one and openssl for the other down below.

Reply:
The authvars TA does not support OSSL yet, and I'm fairly sure there will need to be a few changes to get OSSL for 64 bit. Once the authvars TA works with OSSL I'll update the imx6/7 makefiles to default to OSSL.